### ISSUE TITLE
Links do not have enough contrast or visual cues

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#1457]
The links (especially in large blocks of text) need better (3x1) visual contrast. They are currently blue but hard to pick out of the surrounding black text.

The standard is to underline hyperlinks. We can use a highlight color (yet to be identified) for hyperlinks but we should also consider displaying an underline when a user hovers over the link or they tab onto the link.

This feedback was provided from the UCOP electronic accessibility review. It was defined as having 'High' priority.

**Comment by johnpinto1** [comment]
@briri @xsrust: I have found a suitable contrast using https://webaim.org/resources/linkcontrastchecker/?fcolor=000000&bcolor=FFFFFF&lcolor=107896
![Selection_059](https://user-images.githubusercontent.com/8876215/61715567-4ac8a680-ad55-11e9-8d57-2c2e8b9ba8d3.png)
The next step is to work out how to apply it to links within TinyMce and other non-editable textareas. Changing the `$color-link-text: $color-blue;` in **app/assets/stylesheets/variables/_colours.scss** is not correct.

**Comment by briri** [comment]
hey @johnpinto1,
You may need to add some state qualifiers to the anchor definition in the `/app/assets/stylesheets/blocks/_html.scss`: https://github.com/DMPRoadmap/roadmap/blob/a08d7516c452f7c9b68a9014e5f38be61f3d2d08/app/assets/stylesheets/blocks/_html.scss#L59

Try changing that to: 
```sass
a, a:hover, a:link, a:visited, a:active {
  color: $color-link-text;
}
a:hover {
  text-decoration: underline;
}
```
That should apply the correct color to most of the site's links. You may need to construct a specific css file for Tinymce. It allows you to define a css it should use: https://www.tiny.cloud/docs/configure/content-appearance/#content_css

**Comment by johnpinto1** [comment]
Screen shots of changes:

![Selection_058](https://user-images.githubusercontent.com/8876215/61960406-48688580-afbd-11e9-98bb-4325dcfaddd4.png)
![Selection_057](https://user-images.githubusercontent.com/8876215/61960407-48688580-afbd-11e9-9da2-c201252387a8.png)

**Comment by johnpinto1** [comment]
Screen shots of changes:

![Selection_064](https://user-images.githubusercontent.com/8876215/61960577-a4cba500-afbd-11e9-9e95-fe9532cb909e.png)
![Selection_063](https://user-images.githubusercontent.com/8876215/61960578-a5643b80-afbd-11e9-97bc-0a94d48199ea.png)
![Selection_062](https://user-images.githubusercontent.com/8876215/61960579-a5643b80-afbd-11e9-9cac-6e3042f5f4cd.png)

**Comment by sjDCC** [comment]
This looks good. I like how links are highlighted too
![link](https://user-images.githubusercontent.com/6169940/65812015-35d71c00-e1b9-11e9-8dca-adc248d1e617.PNG)
![link2](https://user-images.githubusercontent.com/6169940/65812016-35d71c00-e1b9-11e9-8c4c-3ef46e5d807f.PNG)

Closing out
